test/hotspot/gtest/logging/test_logDecorations.cpp

```diff
@@ -32,12 +32,13 @@ static const LogTagSet& tagset = LogTagSetMapping<LOG_TAGS(logging, safepoint)>:
 static const LogDecorators default_decorators;
 
 TEST_VM(LogDecorations, level) {
+  char buf[1024];
```

Comment:
Minor nit: in `LogFileStreamOutput::write_decorations()` you are using a temporary buffer of 255 charaters so you should use the same buffer size for testing. Ideally you can make that a constant or macro which can be reused here in the tests.